cli/package.json

```diff
@@ -65,6 +65,7 @@
     "node-fetch": "^2.6.0",
     "open": "^6.4.0",
     "prompts": "^2.2.1",
+    "request": "^2.88.0",
```

Comment:
Possible to use `node-fetch` which is already installed anyway to get the result as promise, also be more [hip](https://github.com/request/request/issues/3142).